src/View/Home/index.html.twig

```diff
@@ -1,6 +1,17 @@
 {% extends 'layout.html.twig' %}
     {% block title %}{{ parent() }}: Home page{% endblock %}
+    {% block stylesheet %}
+        {{ parent() }}
+        <link rel="stylesheet" href="/assets/css/home/hero.css">
+        <link rel="stylesheet" href="/assets/css/home/formation.css">
+        <link rel="stylesheet" href="/assets/css/home/card-dis.css">
+        <link rel="stylesheet" href="/assets/css/home/intro-main.css">
+    {% endblock %}
 {% block content %}
+    {% include 'components/home/hero.html.twig' %}
+    {% include 'components/home/formation.html.twig' %}
+    {% include 'components/home/intro-main.html.twig' %}
+
     <h1>Welcome to Oxygen School !</h1>
```

Comment:
does this H1 still have its place here ? you already have one in your "hero" component

Reply:
I will remove this line